### DISCUSSION TITLE
security documentation page on readme leads to a 404

### DISCUSSION THREAD

**Discussion by AsteroidOrangeJuice** [authelia/authelia#6769] [Documentation Feedback] [answered]
From https://github.com/authelia/authelia:

"For more information about [security](https://www.authelia.com/information/security/) related matters, please read [the documentation](https://www.authelia.com/information/security/)."

But: 

https://www.authelia.com/information/security/ returns 

"Page not found :(
The page you are looking for doesn't exist or has been moved. If you followed a link to get to this page please report it if it was on this site, the official GitHub repository, or the logs on the latest version."

**Comment by glime-ai** [comment]
Please note this response was created using generative AI leveraging previously available information from the repository. This information may be helpful however it is our strong recommendation that you carefully consider the information before blindly applying it.

---

It appears there might be a mistake in the URL provided for information about security-related matters on Authelia's GitHub page. The correct links to learn more about Authelia's security policies and practices are:

1. For an overview of Authelia's security features and their approach to security, you can visit the security introduction page: [Security Introduction](https://www.authelia.com/overview/security/introduction)
2. To understand Authelia's security policy, including how they handle coordinated vulnerability disclosure, you can visit the security policy page: [Security Policy](https://www.authelia.com/policies/security)
3. For general information or to contact Authelia regarding security vulnerabilities or other security-related inquiries, you can refer to the contact page and specifically view the security section: [Contact - Security](https://www.authelia.com/information/contact)

It's important to follow the guidelines provided by Authelia for reporting security vulnerabilities, which emphasize contacting them privately and not disclosing the issue publicly until it has been addressed.

If you were following a link from the official GitHub repository or any official Authelia documentation that led to a "Page not found" error, it would be helpful to report this issue so it can be corrected. You can do this by contacting Authelia directly through one of their [official contact methods](https://www.authelia.com/information/contact) or by opening an issue in the [Authelia GitHub repository](https://github.com/authelia/authelia/issues) if it pertains to a documentation error.

This comment was generated by [Glime](https://glimelab.ai/?utm_source=6fsHkcyufjQLaB&utm_medium=github).

**Comment by AsteroidOrangeJuice** [comment]
I was going to submit this as an actual issue, but this bullet point made me decide to do a discussion instead:

Please only report bugs with the Authelia application itself. Issues which match one of the following criteria should not be reported here (for the purpose of keeping the issue tracking clear and concise) but should be a [discussion](https://github.com/authelia/authelia/discussions/new/choose) instead:
...
Mistakes in the documentation.
...

**Reply by james-d-elliott** [reply, marked as answer]
Thanks for this, will fix it up or you're welcome to, it's meant to be https://www.authelia.com/policies/security/ we consolidated a lot of things and that was one of them.
